src/components/ui/upload/PreviewFile.tsx

```diff
@@ -21,19 +22,19 @@ const PreviewFile = ({ file, allowDelete = false, onDelete }: Props) => {
 
   return (
     <div className="relative">
-      <img
+      <Image
         className="empty-box object-cover"
         src={imageUrl}
         alt={t('file.preview')}
       />
       {allowDelete && (
-        <button
+        <Button
           onClick={onDelete}
           aria-label={t('button.delete_file')}
           className="absolute bottom-0 right-0 bg-gray-900 bg-opacity-50 text-white w-full flex justify-center items-center py-2 hover:bg-opacity-80 transition-colors duration-200 delay-100"
         >
           <FaRegTrashCan className="w-5 h-5" />
```

Comment:
Ik zou de icon in een Icon component wrappen.

Reply:
Hier heb ik even aparte issue voor gemaakt, aangezien er vast meer icons zijn waar dit bij zou kunnen. :)